dependabot/dependency-tree.txt

```diff
@@ -49,7 +49,7 @@ ctia:ctia:jar:1.1.1-SNAPSHOT
 +- metosin:schema-tools:jar:0.12.2:compile
 +- threatgrid:flanders:jar:0.1.23:compile
 |  \- org.clojure:core.match:jar:0.3.0:compile
-+- threatgrid:ctim:jar:1.1.11:compile
++- threatgrid:ctim:jar:eb3169dfe544ee22358eccdfe7506b3e1a905224:compile
```

Comment:
Non-release jar version?

Reply:
That would be https://github.com/threatgrid/ctim/pull/371